packages/auth-server/abi/nft-quest/paymaster.ts

```diff
@@ -0,0 +1,351 @@
+import type { AddressByChain } from "../types";
+
+export const addressByChain: AddressByChain = {
```

Comment:
Why split all addresses to different files? Now if you need to change an address or just find them you'll have to go through all these files..